lighthouse-core/test/results/sample_v2.json

```diff
@@ -1716,6 +1716,7 @@
       "description": "This is the element that was identified as the Largest Contentful Paint. [Learn More](https://web.dev/lighthouse-largest-contentful-paint)",
       "score": null,
       "scoreDisplayMode": "informative",
+      "displayValue": "0 elements found",
```

Comment:
we want this to be 1 fyi.

try adding a fake trace event to `lighthouse-core/test/results/artifacts/defaultPass.trace.json`. search for "_comment" to see an example of when we did this in the past. this trace was collected awhile ago and we don't really want to wholesale replace it, so we have just been tweaking it

Reply:
Can `lighthouse-core/test/results/artifacts/artifacts.json` be updated like that as well? There's already and LCP event in the trace but it's missing a nodeId, but I'm not sure adding a nodeId would be helpful given the nature of how they work.

Reply:
yup

Reply:
`lcp` needs to be updated in the artifacts.json to `largest-contentful-paint`, that's probably enough.

you're right, just adding the nodeid to the trace event isn't necessary, just need the artifacts to be good.